packages/site-demo/content/product/patterns/filter-and sort/index.mdx

```diff
@@ -4,48 +4,53 @@
 
 ## Anatomy
 
-Filter and sort patterns has three sections: A text filter, trigger filters, and a sorting selector. Every zone combine and is optional.
+Filter and sort patterns have three sections: A text filter, trigger filters, and a sorting selector. Sections are optional and combine.
```

Comment:
Wouldn't it be `combined` instead of `combine`?

Reply:
they combine, they work together. User can use text filter on top of a trigger filter and a sorting option

Reply:
Ok then :slightly_smiling_face: